docs/admin/multiple-zones.md

```diff
@@ -0,0 +1,280 @@
+---
+---
+
+## Introduction
+
+Kubernetes 1.2 adds support for running a single cluster in multiple zones.
```

Comment:
I think GCE refers to them as zones, so I was trying to not be AWS specific.  I wrote this:

> Kubernetes 1.2 adds support for running a single cluster in multiple failure zones
(GCE calls them simply "zones", AWS calls them "availability zones", here we'll refer to them as "zones").